requirements.txt

```diff
@@ -0,0 +1,7 @@
+coverage==7.5.2
```

Comment:
You are using coverage only in your workflow, which I suggested to change. If you do so, you won't need this module in your project.

Reply:
I am not sure what you want here. In the workflow, a call requirement.txt is installed. So, they need to be defined.

Reply:
After the discussion we got on the same page. So this is solved by now.